sdk/docs/storage/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
@danewalton looks like we are missing the LICENSE file for `docs/iot/`
Maybe we should add it.

We have one at the root of the repo already, not sure what the protocol is. @RickWinter or @weshaggard do you know?

Reply:
In general I prefer to keep one license file per-repo. However in some languages the LICENSE file must live next to the source in the repo and be part of the package. For c given we ship as one unit I would stick with a single one in the root.

Reply:
Thank you Wes, I'll remove the duplicate license file.

Reply:
#1981